pobi.md

```diff
@@ -54,74 +54,73 @@ Note that the value of L1_Proof_Height is less than L1_Block_Height.
 
-Ethereum builds on top of this model to handle the complexities of Turing complete smart contract execution, by introducing a notion of _gas_ and of _gas price_, but the high level mechanics remain the same.
+Ethereum builds on top of this model to handle the complexities of Turing complete smart contract execution by introducing a notion of _gas_ and of _gas price_, but the high-level mechanics remain the same.
 
-Compared to a typical L1 protocol, there is an additional complexity to consider. In a L1 like Bitcoin or Ethereum, once a node gossips a valid block, all the other nodes are incentivised to use it as a parent, because they know everyone will do that as well. In a L2 decentralised protocol like POBI, there is an additional step, which is the publication of the rollup to L1, which can fail for multiple reasons. Furthermore, the incentive design must also consider the problem of front-running the actual rollup. For a rollup to be final, it has to be added to a L1 block, which is where a L1 miner or staker can attempt to claim the reward that rightfully belongs to a different L2 node.
+Compared to a typical L1 protocol, there is an additional complexity to consider. In a L1 like Bitcoin or Ethereum, once a node gossips a valid block, all the other nodes are incentivised to use it as a parent, because they know everyone does that too. In a L2 decentralised protocol like POBI, there is an additional step: the publication of the rollup to L1, which can fail for multiple reasons. Furthermore, the incentive design must also consider the problem of front-running the actual rollup. For a rollup to be final, it has to be added to a L1 block, which is where a L1 miner or staker can attempt to claim the reward that rightfully belongs to a different L2 node.
 
-The high level goal is to keep the system functioning as smoothly as possible, and be resistant to random failures or malicious behaviour, while not penalising Obscuro nodes for not being available. Obscuro introduces the concept of _claiming rewards_ independently of the actual canonical rollup chain. The great advantage is increased flexibility in aligning incentives, at the cost of increased complexity. Rewards can be awarded in full, split between aggregators or just enough to cover the cost of gas. The cost of publishing a rollup will never exceed twice the gas cost.
+The high-level goal is to keep the system functioning as smoothly as possible and resist random failures or malicious behaviour while not penalising Obscuro nodes for not being available. Obscuro introduces the concept of _claiming rewards_ independently of the actual canonical rollup chain. The great advantage is increased flexibility in aligning incentives at the cost of increased complexity. Rewards can be awarded in full, split between aggregators or just enough to cover the cost of gas. The cost of publishing a rollup will never exceed twice the gas cost.
```

Comment:
"The cost of publishing a rollup will never exceed twice the gas cost." << does this need to be reflected in the fees section / equation?